beta-component-promotions/2021.06-promotions/2021.06-promotions.md

```diff
@@ -0,0 +1,42 @@
+# PatternFly Beta Component Promotion Candidates
+## For Release 2021.08
+### [Banner](https://www.patternfly.org/v4/components/banner)
+A banner is a one-line, full color, full width container that can be used to communicate short snippets of information to users. Banners are un-intrusive, non-dismissible, and only support link buttons.
+
+![banner](./img/banner.png)
+
+### [Button: inoperableEvents](https://www.patternfly.org/v4/components/button)
+Button prop. An array of events to prevent when the button is in an aria-disabled state.
```

Comment:
What does this mean? I don't want to suggest any edits unless I know the meaning so that I don't sacrifice accuracy at all :)

Reply:
:) This is a hard one...
`inoperableEvents` is a prop the consumer can pass to the `Button` component. It is an array of strings which represent the html events which should be prevented whenever the button is in an `aria-disabled` state. I don't know the details of why this was added, I guess there are probably accessibility related reasons to use the `aria-disabled` attribute, but there are cases when it doesn't exactly behave the way the consumer expects, so we have given them an extra way to control the button's behavior in this case.

Reply:
Ah OK, that makes sense. Thanks!